src/app/api/roles/[id]/route.ts

```
@@ -57,36 +71,119 @@ export async function PATCH(
     // Await params before accessing its properties
     const { id } = await params;
     
+    const body = await request.json();
+    const { id } = await params;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'id'. Consider to delete it or rename it.

'id' is defined here:


(lint/suspicious/noRedeclare)